pkgs/top-level/python-packages.nix

```diff
@@ -3846,36 +3850,17 @@ in {
     };
   };
 
+  colorama = callPackage ../development/python-modules/colorama { };
 
-  colorama = buildPythonPackage rec {
-    name = "colorama-${version}";
+  colorama37 = self.colorama.overrideAttrs (old: rec {
```

Comment:
- only a single version of a package in `pythonPackages`. If you really need another version, include it via a `let ... in` expression. Only if multiple packages need this extra version, we introduce it globally.
- when appending a version number, its supposed to be colorama_3_7 to avoid ambiguation.